packages/framework/collabspaces/src/test/collabSpaces.spec.ts

```diff
@@ -309,6 +310,104 @@ describe("Temporal Collab Spaces", () => {
 		assert(value2?.value === initialValue, "value was preserved correctly");
 	});
 
+	describe("Reverse Mapping tests", () => {
+		async function getRowAndColIdsFromChannelId(
+			collabSpace: IEfficientMatrix & IEfficientMatrixTest,
+			row: number,
+			col: number,
+		): Promise<{ rowId: string; colId: string }> {
+			const result = await collabSpace.getCellChannelIdDebugInfo(row, col);
```

Comment:
I'd suggest to do this kind of parsing in one place (inside of mapChannelToCell()) and simply for getCellChannelIdDebugInfo or other API to return all the required info. That way it's a bit easier to change it if we need to in the future